models/scripts/SentimentAnalysis.py

```diff
@@ -0,0 +1,35 @@
+from tabpy_tools.client import Client
+import pandas as pd
+from textblob import TextBlob
+from nltk.sentiment import SentimentIntensityAnalyzer
+import sys
+
+
+def SentimentAnalysis(_arg1, library='nltk'):
+    assert (type(_arg1[0]) is str)
+
+    library = library.lower()
+    supportedLibraries = {'nltk', 'textblob'}
+
+    assert (library in supportedLibraries)
+    scores = []
+    if library == 'nltk':
+        sid = SentimentIntensityAnalyzer()
+        for text in _arg1:
+            sentimentResults = sid.polarity_scores(text)
+            score = sentimentResults['compound']
+            scores.append(score)
+    elif libary == 'textblob':
+        for text in _arg1:
+            currScore = TextBlob(text)
+            scores.append(currScore.sentiment.polarity)
+    return scores
+
+
+if __name__ == '__main__':
+    port = sys.argv[1]
+    connection = Client('http://localhost:' + port + '/')
```

Comment:
These will need to be updated once tools supports auth.

Reply:
+1

Reply:
added a to do comment to both PCA and SentimentAnalysis